site/en/docs/android/custom-tabs/integration-guide/index.md

```diff
@@ -48,29 +49,84 @@ customTabsIntent.launchUrl(this, Uri.parse(url));
+```groovy
+implementation 'androidx.browser:browser:1.5.0-alpha01'
+```
 
-The snippet below changes the launch height of the Custom Tab. `CustomTabsIntent.Builder` has a new method `setInitialActivityHeightPx()` that allows you to define the initial launch height in pixels. As a prerequisite to using Custom Tabs, you will need to [connect to the Custom Tabs Service](/docs/android/custom-tabs/integration-guide/#connect-to-the-custom-tabs-service) or launch the custom tabs intent using [Activity#startActivityForResult()](/reference/android/app/Activity#startActivityForResult(android.content.Intent,%20int)).
+To turn a Custom Tab into a partial Custom Tab, define the initial launch height in pixels by calling `CustomTabsIntent.Builder#setInitialActivityHeightPx()`. Furthermore, you need to either:
 
-There are properties that do not work when building an Intent for partial Custom Tabs:
+1. [start a new browser session via a `CustomTabsServiceConnection`](/docs/android/custom-tabs/integration-guide/#connect-to-the-custom-tabs-service) and pass it to the Custom Tabs intent or
+2. start the Custom Tab activity via [`startActivityForResult`](https://developer.android.com/reference/android/app/Activity#startActivityForResult(android.content.Intent,%20int)).
 
--   `CustomTabColorScheme.navigationBarColor`
--   `CustomTabColorScheme.navigationBarDividerColor`
+Starting a new browser sessions is the recommended approach to launching a partial Custom Tag, as it enables you to [warm up the browser process](#warm-up-the-browser-process) and [pre-render webpages](/docs/android/custom-tabs/best-practices/#pre-render-content). If you don't need these features, using `startActivityForResult()` is the easier way to launch a partial custom tab.
 
-Custom Tabs will inherit the host app's color scheme for the UI properties above. You will be responsible for ensuring visual consistency for these properties before launching a Custom Tab.
+{% Aside 'gotchas' %}
+* Specifying the initial activity height will not have an effect if the default browser does not support resizing the Custom Tab. In this case, the intent extra will be ignored and the Custom Tab will span the complete display height.
 
-These new APIs are available from Chrome 107 and onwards.
+* Custom Tabs will inherit the host app's color scheme for the user interface properties above. You will be responsible for ensuring visual consistency for these properties before launching a Custom Tab. This means the `CustomTabColorScheme.navigationBarColor` and `CustomTabColorScheme.navigationBarDividerColor` properties do not work when building an intent for a Custom Tab.
```

Comment:
Could be because I'm new the review UI, but it looks like the lines listing the properties above got deleted, which is fine. We can just list them here but need to rephrase.

Reply:
Good catch! Added "for a partial Custom Tab"